src/components/Screen/index.js

```diff
@@ -71,6 +71,11 @@ export class Screen extends Component {
 							<span>{agent.email}</span>
 						</Header.SubTitle>
 					)}
+					{agent.phone && (
```

Comment:
@madhavrathi, you need to check if the agent property is assigned before reading the `agent.phone`:

`agent && agent.phone`.